src/main/kotlin/com/getkeepsafe/dexcount/OutputFormat.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015-2016 KeepSafe Software
```

Comment:
I just used the same copyright for all the files, the 2016 version. Did you want me to add it back?